### PR TITLE
Fixed riswhois (RIPE) support

### PR DESCRIPTION
**Pull request by matsduf**
Resolves issue #833

### CODE REVIEW

**Review by vlevigneron — APPROVED** [review]

**Review by pnax — COMMENTED** [review]
Maybe there should be an automated test to check that everything works fine with the RIPE profile.

File: lib/Zonemaster/Engine/ASNLookup.pm [hunk 1]
```diff
@@ -137,13 +137,18 @@ sub _ripe_asn_lookup {
...
-            return \@asns, undef, q{}, q{ERROR_ASN_DATABASE};
-        }
-        else {
-            next;
-        }
+        my $socket;
+        unless( $socket = IO::Socket::INET->new( PeerAddr => $db_source->name->string, 
+                                                 PeerPort => q{43}, 
+                                                 Proto => q{tcp} ) 
+            ) {
```
[the remark this answers is not in the document]
Comment (matsduf) [reply lib/Zonemaster/Engine/ASNLookup.pm]: Same, same, but I changed the code as you suggested.

**Comment by matsduf** [comment]
> Maybe there should be an automated test to check that everything works fine with the RIPE profile.

One complexity is that you do not want automated tests that are dependent on remote connection. We have built-in solution for canned DNS queries, but this something different. But it is a good suggestion. I'll create an issue.

**Review by pnax — APPROVED** [review]

**Review by mattias-p — APPROVED** [review]

**Comment by matsduf** [comment]
@vlevigneron, could you review after my change?

**Review by vlevigneron — APPROVED** [review]

**Pull request merged**